samples/net/rpl-node/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(QEMU_NET_STACK 1)
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(NONE)
+
+target_sources(app PRIVATE src/main.c)
+
+if(CONFIG_NET_L2_IEEE802154)
```

Comment:
CONFIG_NET_L2_IEEE802154 is always needed for this sample

Reply:
Fixed.